### ISSUE TITLE
A better way to import external library

### ISSUE THREAD

**Issue by xcv58** [xcv58/Custom-JavaScript-for-Websites-2#1]
Currently, we can only use the `your own external scripts` window to import external library. It's not good because we need maintain both injected scripts and library import scripts.

There are two possible ways:
1. import from scripts
2. link to cdnjs.com and import by click

**Comment by xcv58** [comment]
We can provide a function like:

```js
import('//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js')
```

to import external library.

**Comment by juloko** [comment]
Why that is not available anymore? This is a Ryan Dahl dream.

**Comment by xcv58** [comment]
> Why that is not available anymore? This is a Ryan Dahl dream.

![image](https://user-images.githubusercontent.com/503123/98496106-aee6d680-21f5-11eb-8f9c-c46dc6f9c7fe.png)


This is still available, you can click "your own external scripts" and include the URL for the JS file.

**Comment by juloko** [comment]
Thanks for the attention.

I saw someone talking about that link should be a CDN, I tried but didn't work.

Must be a specific CDN?

Follow my CDN link in below:
[https://cdn.jsdelivr.net/gh/juloko/BrotherBet-Bot/jobs/bet365Futsal.js](url)

Shoud  `console.log("There isn't any games.")`. But nothing happens.

In documentation said to put a `;` at the and but is not possible the field remove than and break a line.


BTW, in script works as well. This is a awesome project. I'd like to contribute to add these links in front-end in a facility way, and implement a global function like this:
```javascript
cjs.import('cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js')
```

**Comment by xcv58** [comment]
Sorry, you need to omit the `https:` part. I tried on github.com and it works:

![image](https://user-images.githubusercontent.com/503123/98625131-ecb23080-22c3-11eb-9fb1-21872e396325.png)


I'll try to make it more clear.

**Comment by xcv58** [comment]
It's caused by: https://github.com/xcv58/Custom-JavaScript-for-Websites-2/blob/master/src/js/run.tsx#L47

**Comment by xcv58** [comment]
Fixed in 21e91a1a2dc9b8a54330781b836101cbfc5be3fd

**Comment by xcv58** [comment]
>BTW, in script works as well. This is a awesome project. I'd like to contribute to add these links in front-end in a facility way, and implement a global function like this:

Please feel free to reopen this issue or open a new one.

This functionality shouldn't hard to implement, we can use a similar way like the `customjsReady ` https://github.com/xcv58/Custom-JavaScript-for-Websites-2/blob/master/src/js/base.tsx#L40
